apps/ledger-live-desktop/tests/specs/speculos/add.account.spec.ts

```diff
@@ -8,7 +8,7 @@ const currencies: Currency[] = [
   Currency.ETH,
   Currency.ETC,
   Currency.XRP,
-  Currency.DOT,
+  //Currency.DOT, //Reactivate after DOT API issue is resolved - TSD-3603
```

Comment:
`//todo: Reactivate after DOT API issue is resolved - TSD-3603` will be easier to find

Reply:
done